tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -44,6 +44,13 @@ Then('the success/error message {string} should be displayed on the webUI', func
     .expect.element('@message').text.to.equal(message)
 })
 
+Then('the following success/error message should be displayed on the webUI', function (message) {
+  return client
+    .page.phoenixPage()
+    .waitForElementVisible('@messages')
+    .expect.element('@messages').text.to.equal(message)
+})
+
```

Comment:
We have similar step ` Then the error message 'File upload failed…' should be displayed on the webUI`
Also @messages xpath only include the title, so it doesn't check the body

Reply:
`@message` xpath inclide the div with `notification-message-title`. So, if we reuse the step `Then the error message 'File upload failed…' should be displayed on the webUI`, we only get the title text of the error message. The `@messages` xpath points to the div which contains the whole message in the error

Reply:
Yup, that's what I am talking about, and the `@message` selector in above code still points to `title-only` div, so fix that

Reply:
Looks like `@messages` do the job. Looks good

Reply:
we should fix the wording of `the success/error message {string} should be displayed on the webUI` otherwise it will be too confusing. I will open an issue for that

Reply:
issue opened https://github.com/owncloud/phoenix/issues/3062 @Talank you could work on it